dotcom-rendering/src/components/SetABTests.importable.tsx

```diff
@@ -62,7 +59,6 @@ export const SetABTests = ({
 			...cypressAbSwitches, // by adding cypress switches below CAPI, we can override any production switch in Cypress
 		},
 		arrayOfTestObjects: tests,
-		ophanRecord,
```

Comment:
This has actually stopped us registering client-side AB tests.

Discovered jointly with @lindseydew.